index.html

```diff
@@ -0,0 +1,116 @@
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+</head>
+<body>
+  <header>
+    <div class="topbar">
+      <ul>
+        <li><a href="#"><i class="bi bi-facebook"></i></a></li>
+        <li><a href="#"><i class="bi bi-twitter"></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <nav>
+      <div class="nav-bar">
+        <a href="index.html"><img src="./images/Logo.png" alt="holi-logo" class="logo"></a>
+        <button class="hambug-menu">
+          <i class="menuIcon material-icons">menu</i>
+          <i class="closeIcon material-icons">close</i>
+        </button>
+        <ul class="mobile-menu">
+          <li><a class="menuItem" href="index.html">Home</a></li>
+          <li><a class="menuItem" href="about.html">About</a></li>
+          <li><a class="menuItem" href="#">Programs</a></li>
+          <li><a class="menuItem" href="#">Gallery</a></li>
+          <li><a class="menuItem" href="#">Come Celebrate</a></li>
+        </ul>
+
+        <ul class="nav-links">
+          <li class="links"><a class="alinks" href="about.html">About</a></li>
+          <li class="links"><a class="alinks" href="#">Programs</a></li>
+          <li class="links"><a class="alinks" href="#">Gallery</a></li>
+          <li class="links"><a class="alinks borderLink" href="#">Come Celebrate</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <section class="headContent">
+    <h1 class="headerTitle">"Have a Happy Holi!"</h1>
+    <h2 class="subHeader">HOLI - "Festival of Colors" or the "Festival of Love"</h2>
+    <p class="headerParagraph">Holi has religious and mythological significance, marking the victory of good over evil and the arrival of spring. Holi is now celebrated globally by people of Indian origin and has become a symbol of India's cultural richness and diversity.</p>
+    <p class="headerDate">2023.03.07 ~ 24</p>
+    <p class="headerLocation">@ Vrindavan, INDIA</p>
+    </section>
+
+  <section class="main">
+    <h2 class="sub">Main Program</h2>
+    <p class="line"></p>
+    <div class="mainDiv">
+      <div>
+        <i class="bi bi-flower1 icon"></i>
+        <span>Flower</span>
+        <p>Holi is celebrated on the full moon day in the Hindu month of Phalguna, which falls in February or March.</p>
+      </div>
+      <div>
+        <i class="bi bi-heart-fill icon"></i>
+        <span>Love</span>
+        <p>Holi is also associated with several legends, such as the Love story of Lord Krishna and Radha.</p>
+      </div>
+      <div>
+        <i class="bi bi-paint-bucket icon"></i>
+        <span>Color</span>
+        <p>People throw colored powder and water on each other, sing and dance to traditional music, and feast on sweet delicacies.</p>
```

Comment:
- [ OPTIONAL ] For this section Kindly make sure to add the background as shown in Cindy's design:
Your design:
![Screenshot from 2023-02-10 14-41-09](https://user-images.githubusercontent.com/97834160/218096042-ab3959f9-be5d-46ad-8a3a-bebb1206b6cd.png)
Cindy's:
![Screenshot from 2023-02-10 04-43-26](https://user-images.githubusercontent.com/97834160/218096265-5014af25-e167-4c45-bddf-d7588107d6d3.png)